src/test/regress/expected/propagate_foreign_servers.out

```diff
@@ -0,0 +1,103 @@
+-- remove node to add later
+SELECT citus_remove_node('localhost', :worker_1_port);
+ citus_remove_node
+---------------------------------------------------------------------
+
+(1 row)
+
+-- create schema, extension and foreign server while the worker is removed
+CREATE SCHEMA test_dependent_schema;
+CREATE EXTENSION postgres_fdw WITH SCHEMA test_dependent_schema;
+CREATE SERVER foreign_server_dependent_schema
+        FOREIGN DATA WRAPPER postgres_fdw
+        OPTIONS (host 'test');
+SELECT 1 FROM citus_add_node('localhost', :worker_1_port);
+ ?column?
+---------------------------------------------------------------------
+        1
+(1 row)
+
+\c - - - :worker_1_port
+-- verify the dependent schema and the foreign server are created on the newly added worker
+SELECT COUNT(*) FROM pg_namespace WHERE nspname = 'test_dependent_schema';
+ count
+---------------------------------------------------------------------
+     1
+(1 row)
+
+SELECT COUNT(*)=1 FROM pg_foreign_server WHERE srvname = 'foreign_server_dependent_schema';
+ ?column?
+---------------------------------------------------------------------
+ t
+(1 row)
+
+\c - - - :master_port
+SET client_min_messages TO ERROR;
+DROP SCHEMA test_dependent_schema CASCADE;
+RESET client_min_messages;
+-- test propagating foreign server creation
+CREATE EXTENSION postgres_fdw;
+CREATE SERVER foreign_server TYPE 'test_type' VERSION 'v1'
+        FOREIGN DATA WRAPPER postgres_fdw
+        OPTIONS (host 'testhost', port '5432', dbname 'testdb');
+SELECT COUNT(*)=1 FROM pg_foreign_server WHERE srvname = 'foreign_server';
+ ?column?
+---------------------------------------------------------------------
+ t
+(1 row)
+
+\c - - - :worker_1_port
+-- verify that the server is created on the worker
+SELECT COUNT(*)=1 FROM pg_foreign_server WHERE srvname = 'foreign_server';
+ ?column?
+---------------------------------------------------------------------
+ t
+(1 row)
+
+\c - - - :master_port
+ALTER SERVER foreign_server OPTIONS (ADD passfile 'to_be_dropped');
+ALTER SERVER foreign_server OPTIONS (SET host 'localhost');
+ALTER SERVER foreign_server OPTIONS (DROP port, DROP dbname);
+ALTER SERVER foreign_server OPTIONS (ADD port :'master_port', dbname 'regression', DROP passfile);
```

Comment:
can you do a quoted parameter: `ADD "port" : ...`

Reply:
I think it's now allowed

Reply:
I can check other FDWs to see if it's possible or not